palinagent/daemon/main.py

```diff
@@ -47,8 +47,28 @@
 try:
     from lxml import etree
 except (ImportError), e:
-    print >> sys.stderr, AgentSetupError("lxml not available: %s" % e)
-    sys.exit(7)
+    # lxml is not installed
```

Comment:
If possible, add the information about the architecture and the installed lxml package in the method **_log_debug_info(logger)**

Reply:
added them